src/js/themes/microvideo.js

```diff
@@ -440,7 +440,7 @@ template.innerHTML = /*html*/`
 </template>
 
 <media-controller
-  style="--_control-bar-place-self:{{controlBarPlace ?? 'unset'}}"
+  style="--_control-bar-place-self:{{controlbarplace ?? 'unset'}}"
```

Comment:
NOTE: Because we use JS to translate attr -> prop, using all lowercase/"smushedcase" here doesn't allow us to use `camelCase` to translate (since there are no delimiters to match against with the new names).

Reply:
We may want to treat this as a separate effort and descope it for this initial round of changes.